packages/cli/src/commands/replay/replay.command.ts

```diff
@@ -98,15 +104,25 @@ export const replayCommandHandler: (
 
   const meticulousSha = await getMeticulousVersion();
 
-  // 3. Load replay assets
+  // 3. If simulationIdForAssets specified then download assets & spin up local server
+  if (simulationIdForAssets) {
+    const result = await serveAssetsFromSimulation(
+      client,
+      simulationIdForAssets
+    );
+    appUrl = result.url;
+    closeServerIfRunning = result.closeServer;
```

Comment:
Not a fan of mutable variables if we could avoid that :thinking:

Reply:
Me neither! I could do:

```ts
const server = simulationIdForAssets ? await serveAssetsFromSimulation(
      client,
      simulationIdForAssets
    ) : undefined;
const urlToRunSimulationAgainst = server ? server.url : appUrl ;

[...] (and replace refs to appUrl with urlToRunSimulationAgainst)

server?.closeServer?.();
```

Reply:
I've refactored, I think this is nicer